src/components/UI/Typography/Typography.tsx

```diff
@@ -34,7 +40,8 @@ export const Typography = ({
     <MantineText
       size={mappedSize}
       style={{ ...variantStyle, ...style }}
-      data-testid="typography-component" 
+      data-testid="typography-component"
+      ta={ta}
```

Comment:
Same thing about all of those changes, do not change stuff like this out of the blue
talk to me first